tests/dragonfly/snapshot_test.py

```diff
@@ -46,6 +51,7 @@ async def test_consistency(df_factory, format: str, seeder_opts: dict):
     Test consistency over a large variety of data with different sizes
     """
     dbfilename = f"dump_{tmp_file_name()}"
+    rm_glob(f"{dbfilename}*")
```

Comment:
You are trying to remove files generated by previous runs of the same test. However, `tmp_file_name()` should generate  unique enough filenames such that they don't clash. So I guess the question is, did you see two runs of the same test sharing the same `dbfilename` and you added this change ?

Reply:
We saw a failure in `test_multidb`. I'm trying to fix other similar failures.
Specifically I agree that the chances of hitting the same filename with `tmp_file_name()` are slim (`52^10`). So perhaps we should just use that all over instead of the random(5000) and deleting files. I'll do that.

Reply:
I though the issue here was that the tests runs repeatedly with different arugments, so the names can clash. We already have a session wide tmp folder that gets deleted after a run

Reply:
@dranikpg This is an interesting question. I see that `test_multidb` fails, which is not a parameterized test.
When running locally, with `--count=X` I can reproduce this, but as you rightly pointed out in GitHub it should run in a session temp folder, and we don't run with `--count` so how can that be?
The failure was 2w ago, so I guess that hasn't changed.

Maybe we use a session-wide logs folder, that's different from where the rdb files are written?
(I can't find this area in the `.github/` files)